testbed/tests/scenarios.go

```diff
@@ -235,3 +237,48 @@ func Scenario1kSPSWithAttrs(t *testing.T, args []string, tests []TestCase, opts
 		})
 	}
 }
+
+// Structure used for TestTraceNoBackend10kSPS.
+// Defines RAM usage range for defined processor type.
+type ProcessorConfig struct {
+	Name string
+	// map of processor types to their config YAML to use.
+	ProcessorConfig     map[string]string
```

Comment:
It is a bit confusing when the name of the field is the same as the struct it is in. Can you come up with a better name?